tools/BatteryModel.py

```diff
@@ -0,0 +1,42 @@
+# Class model of a standard LiPo battery
+
+import sys
+import pyulog as ulog
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.optimize import curve_fit
+
+class BatteryModel:
+    def __init__(self, file, nominal_voltage: float = 11.1, nominal_mAh: float = 5100.0, cells: int = 3):
```

Comment:
You can remove all inputs to init, none of it is used. In fact, just remove the `__init__(self, ...)` method all together